Apps/Common/test/unit/Services/PersonalAccountServiceTests.cs

```diff
@@ -82,31 +79,6 @@ public void ShouldGetPatientAccountUseCache()
             Assert.Equal(id, actualResult.ResourcePayload?.Id);
         }
 
-        /// <summary>
-        /// Get Personal Account returns ApiException.
-        /// </summary>
-        [Fact]
-        public void ShouldGetPatientAccountReturnsApiException()
```

Comment:
Where is the unit test to handle ApiException?. Won't this have an effect on code coverage?  Looks like only HttpRequestException is being handled.

Reply:
See my previous comment. There shouldn't be any effect on code coverage. This is the same as the test changes that Stephen checked in earlier, one test for an exception.